afs/media/js/views/components/reports/scenes/substance.js

```diff
@@ -27,42 +27,107 @@ define(['underscore', 'knockout', 'arches', 'utils/report','bindings/datatable']
             Object.assign(self.dataConfig, params.dataConfig || {});
 
             // if params.compiled is set and true, the user has compiled their own data.  Use as is.
-            if(params?.compiled){
-                self.dimensions(params.data.dimensions);
-            } else {
-                let dimensionData = self.getRawNodeValue(params.data(), self.dataConfig.dimension);
-                
-                if(dimensionData) {
-                    if(dimensionData.length === undefined){
-                        dimensionData = [dimensionData];
-                    } 
+            if(self.dataConfig.dimension){
+                if(params?.compiled){
```

Comment:
Seems like params.compiled should be outside of this if check - if params.compiled is set, the user has compile/provide to set _all_ of the data on their own, including dimension and timespan.

Reply:
Removed